lib/genesis/genesis.go

```diff
@@ -41,14 +41,20 @@ type Data struct {
 	ID                 string
 	ChainType          string
 	Bootnodes          [][]byte
-	TelemetryEndpoints []interface{}
+	TelemetryEndpoints []TelemetryEndpoint
```

Comment:
Change this to pointer.
`TelemetryEndpoints []*TelemetryEndpoint`

Reply:
done.